microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -131,6 +135,8 @@ If you aren't able to use the emulators and you don't have access to real device
 
 The following list is a sample of cloud-based emulators and testing sites. Review the descriptions for features or capabilities to consider when selecting a testing site. Conduct your own search to find the best cloud-based emulator for your needs.
 
+<!-- todo: delete section/list, use MDN instead -->
```

Comment:
delete list of other companies' products from section "Cloud-based emulators and simulators", & link to MDN instead?
@zoherghadyali @bwalderman 

Rendered article section: 
pending

Reply:
This list has gotten a little stale. As far as I can tell, the "Mobileum" mentioned here no longer exists and an unrelated company now owns the linked page. Probably best to replace this section.

This is the closest MDN equivalent I can find: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Testing_strategies

It's mostly about setting up your own cross-device testing solution, but it does mention some of the commercial cloud-based tools we're currently listing here.

Reply:
replaced lead-in and list by
```
See [Emulators](https://developer.mozilla.org/docs/Learn/Tools_and_testing/
Cross_browser_testing/Testing_strategies#emulators) in _Strategies for 
carrying out testing_ at MDN.
```
https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Testing_strategies#emulators